docs: add shell completion to nav.js (#1357)
Bearer/bearer: docs/guides/shell-completion.md

title: Enable Completion Script
---

# Enabling shell completions

> Completions make it easier to use bearer on the command line. In this guide we will cover how to set everything up.

Reviewer: ```suggestion
Completions make it easier to use Bearer on the command line. In this guide we will cover how to set everything up.
```

## Discover your current shell

```bash
$ echo $SHELL
/bin/zsh
```

> In our example it's zsh, but will be vary depend on your $SHELL.

Reviewer: ```suggestion
In our example it's zsh, but this will vary depending on your $SHELL.
```

## Test the completion scripts

We currently support `zsh` `bash` and `fish`. To load completions in your current shell session you can run the following:

```bash
source <(bearer completion zsh); compdef _bearer bearer
```

> Now you can test completions:

Reviewer: ```suggestion
Now you can test completions. Type `bearer` into your terminal and hit the tab button on your keyboard:
```

Reviewer 2: will use this but remove the shell example

```bash
bearer [tab]
```

**Note:** if completions are not already enabled in your environment you will need
to enable it. In zsh you can execute the following once:

```bash
echo "autoload -U compinit; compinit" >> ~/.zshrc
```

## Final setup

> To load completions for every new session you can add the test command to your `~/.zshrc` or similar. Alternatively if you are using zsh you can execute the following once:

Reviewer: It may be useful to repeat the test command here (sorry - suggestion markdown is getting confused with the "```" but you see the idea)

```suggestion
To load completions for every new session, you can add the test command to your `~/.zshrc` or similar: 

`bash
  source <(bearer completion zsh); compdef _bearer bearer
`

Alternatively, if you are using zsh, you can execute the following once:
```

Reviewer 2: Think its overkill in this case.
